packages/plugin-page-view-v1-enrichment/src/page-view-v1-enrichment.ts

```diff
@@ -0,0 +1,49 @@
+import { BrowserConfig, EnrichmentPlugin, Event } from '@amplitude/analytics-types';
+import { CreatePageViewV1EnrichmentPlugin } from './typings/page-view-v1-enrichment-plugin';
+
+export const pageViewV1EnrichmentPlugin: CreatePageViewV1EnrichmentPlugin = function () {
+  const plugin: EnrichmentPlugin = {
+    name: '@amplitude/plugin-page-view-v1-enrichment-browser',
+    type: 'enrichment',
+
+    setup: async function (config: BrowserConfig) {
+      config.loggerProvider.log('Installing @amplitude/plugin-page-view-v1-enrichment-browser.');
+    },
+
+    execute: async (event: Event) => {
+      if (event.event_type == '[Amplitude] Page Viewed') {
+        const {
+          '[Amplitude] Page Domain': page_domain,
+          '[Amplitude] Page Location': page_location,
+          '[Amplitude] Page Path': page_path,
+          '[Amplitude] Page Title': page_title,
+          '[Amplitude] Page URL': page_url,
+        } = event.event_properties || {};
+
+        event = {
+          ...event,
+          event_type: 'Page View',
+          event_properties: {
+            ...event.event_properties,
+            /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+            page_domain: page_domain ?? '',
+            page_location: page_location ?? '',
+            page_path: page_path ?? '',
+            page_title: page_title ?? '',
+            page_url: page_url ?? '',
+            /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+          },
+        };
+
+        delete event.event_properties?.['[Amplitude] Page Domain'];
```

Comment:
The current test coverage didn't reach 100% because of those lines. 

<img width="541" alt="Screen Shot 2023-08-08 at 12 39 34 PM" src="https://github.com/amplitude/Amplitude-TypeScript/assets/13028329/7c68a745-c570-45a6-9d48-d48636393522">

However, I tried to add the cases, and none of it will increase the test coverage. 
- event_properties is missing in the event
- event_properties is undefined in the event
- has event_properties and there is no key for what we are deleting here. 
- expect the delete call won't throw any error. 

Do you have any idea on this?

Reply:
cc: @Mercy811 @justin-fiedler

Reply:
Hmm, I have had similar issues before. In my case I added a test like `event_properties is undefined in the event` to get the error to go away.

In this case maybe it is to try to delete a prop off a potentially undefined field? Maybe this could work instead?
```
 if('event_properties' in event) {
  delete event.event_properties.['...']
}
```

Reply:
That does work. I spent too much time on this.  I feel that's the jest test issue.
Very much appreciate your recommendation. It saved me.